### ISSUE TITLE
[BITV] Share link row should be fully focusable / clickable to open menu

### ISSUE THREAD

**Issue by Pytal** [nextcloud/server#36111]
![image](https://user-images.githubusercontent.com/24800714/210678789-f0cce99f-985c-4bb9-98c7-696901682334.png)

Highlighting the entire row does not seem necessary but if it is then we may have to redesign all the rows. Does the current design with only the highlighted button area itself being activatable work accessibility-wise @michaelnissenbaum? Or would the whole row have to be activatable?

**Comment by michaelnissenbaum** [comment]
It's not necessary to highlight the whole row - only highlighting the button is enough. You just need to keep in mind, that keyboard focus has to be clearly visible and meet the contrast requirements. I'd recommend to use a clearly visible frame around the button.

**Comment by Pytal** [comment]
Got it @michaelnissenbaum!

Button focus border present on `master` and `stable25`
